### ISSUE TITLE
add commands to the command palette 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#164513]
I was surprised to find that there is no `Update VS Code` action available in the command palette. 

When accessibility support is enabled, should we provide more actions, like that one, as an alternative/ faster way to access menu based actions?

**Comment by meganrogge** [comment]
@isidorn thoughts appreciated 🙏

**Comment by isidorn** [comment]
@meganrogge I like this idea. Overall I think all our commands should be in the command palette. And that if accessibility mode is enabled or not should not influence this.
Having commands in the command palette helps every user - they can add shortcuts etc...

As for updates, I noticed there is an action "Check for Updates". Does that not do what you would expect?

**Comment by meganrogge** [comment]
Yes that's a good pt re accessibility mode.

I do not see a `Check for updates` action...

<img width="764" alt="Screen Shot 2022-10-25 at 9 49 27 AM" src="https://user-images.githubusercontent.com/29464607/197791345-53ce2302-aaf5-4a96-95a4-42ce12bc3582.png">

**Comment by isidorn** [comment]
Hmm I see it 
![Screenshot 2022-10-25 at 16 29 03](https://user-images.githubusercontent.com/1926584/197801509-b0a1977a-8fb6-4681-b9fe-7e6227ae5eac.png)

Here's where the action is registered https://github.com/microsoft/vscode/blob/929c645c9a3727541c8fb30f4d62765f5c672ff9/src/vs/workbench/contrib/update/browser/update.contribution.ts#L85

So it seems like you have an update pending and it is not shown.
But I agree, we should clean this up. It should just be one action that is always there - to apply the update.

**Comment by meganrogge** [comment]
let's use this issue to track any others that we think should be added.

**Comment by meganrogge** [comment]
This exists already

![Image](https://user-images.githubusercontent.com/29464607/200089600-fd21e166-554c-431f-b114-511e42dcc9ec.png)
